torch/csrc/jit/passes/onnx.cpp

```diff
@@ -343,16 +343,17 @@ void NodeToONNX(
     if (func) {
       pyobj = func->get();
     }
-    if (!py::hasattr(pyobj, "symbolic")) {
+
+    py::object is_registered_op = onnx_registry.attr("is_registered_op")("prim_PythonOp", "", 9);
```

Comment:
I may be off base here, but do we need to check whether op is registered only for Opset 9?

Reply:
Updated to check for opset version under export